source/aperture.f90

```diff
@@ -501,13 +501,13 @@ subroutine aperture_backTrackingInit
   i=1
   ix=ic(i)-nblo
   if( ix.lt.0 ) then
-    write(lout,"(a)") "APER> ERROR Impossible to properly initialise backtracking:"
-    write(lout,"(a)") "APER>       first element of lattice structure is not a single element"
+    write(lerr,"(a)") "APER> ERROR Impossible to properly initialise backtracking:"
+    write(lerr,"(a)") "APER>       first element of lattice structure is not a single element"
     call prror(-1)
```

Comment:
prror takes the value as an optional value. It defaults to -1 anyway, but I thought I'd keep the actual option in case we do want to differentiate error codes. May be useful to separate input errors vs. runtime errors for instance.

Reply:
`grep -ir wtf .` in sixtrack folder only returns stuff in `lib`
Where is the WTF?